deploy/test/test_update_alerts_websocket_v2.py

```
@@ -0,0 +1,597 @@
+    mock_get_regions.return_value = (districts)
+
+    mock_get_cache_data.return_value = [[1, 1700000000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], 
+                        [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], 
+                        [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], 
+                        [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], 
+                        [0, 1645674000], [0, 1645674000]
+                        ]
+
+    await update_alerts_websocket_v2(mock_mc, run_once=True)
+
+    expected_result = [[1, 1700000000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], [0, 1645674000], 
```

Ruff (0.8.2)

Local variable `expected_result` is assigned to but never used

Remove assignment to unused variable `expected_result`

(F841)